CanSignals = {
	"targetContactorsStatus": None,
	"evConnectionState ": None,
	"chargingState": None,
	"targetVoltage": None,
	"targetCurrent": None,
	"batteryStateOfCharge": None,
	"cableCheckVoltage ": None,
	"vehicleId": None,
	"chargeMode": None,
	"evMinPower": None,
	"evMinCurrent": None,
	"evMinVoltage": None,
	"evMaxCurent": None,
	"evMaxPower": None,
	"evMaxVoltage": None,
	"chargingProfileMaxPowerLimit": None,
	"timeToFullSoc": None,
	"CHRGR_iAcAct": None,
	"CHRGR_iAcAct2": None,
	"CHRGR_iAcAct3": None,
	"CRC_OBC_AcCurrents": None,
	"mc_OBC_AcCurrents": None,
	"CHRGR_uAcActPha1": None,
	"CHRGR_uActPha2": None,
	"CHRGR_uActPha3": None,
	"CRC_OBC_AcVoltages": None,
	"mc_OBC_AcVoltages": None,
	"CHRGR_iDcAct": None,
	"CHRGR_stOperAct": None,
	"CHRGR_uDcAct": None,
	"CRC_OBC_Status": None,
	"mc_OBC_Status": None,
	"CHRGR_stErr": None,
	"CHRGR_stHvIntlck": None,
	"CRC_OBC_WarnInfo": None,
	"DC1_bReqMIL": None,
	"DC1_stT30c": None,
	"mc_OBC_WarnInfo": None,
}